### ISSUE TITLE
Can't work with enclosejs

### ISSUE THREAD

**Issue by getmeaway** [jfromaniello/winser#38]
hi, 
winser run well with my nodejs app (i run "node server.js") but when i package my app with [enclosejs](enclosejs.com) into an .exe file, winser is not running well. 
It searchs for the *package.json* file and it can not find it cause i have only an exe file.
Is there a way to work around this problem??

**Comment by baconface** [comment]
No, you need to run the .exe as a service without winser. https://stackoverflow.com/questions/3582108/create-windows-service-from-executable
